src/main/java/com/yahoo/bullet/postaggregations/PostStrategy.java

```diff
@@ -0,0 +1,19 @@
+/*
+ *  Copyright 2018, Oath Inc.
+ *  Licensed under the terms of the Apache License, Version 2.0.
+ *  See the LICENSE file associated with the project for terms.
+ */
+package com.yahoo.bullet.postaggregations;
+
+import com.yahoo.bullet.common.Initializable;
+import com.yahoo.bullet.result.Clip;
+
+public interface PostStrategy extends Initializable {
```

Comment:
Does this need to extend Initializable? It looks like both implementing classes just return Optional.empty() for initialize(), despite the fact that they have member variables that can be initialized.

If we are assuming that the member variables of the PostStrategy objects will be initialized somewhere else (which looks like a safe assumption for now), then maybe we can just remove the initialize() method in the PostStrategy objects?

Reply:
Added a default initialize() in the interface. Subclass can override it if it needs initialize its own variables.